administrator/components/com_content/Helper/PreviewHelper.php

```diff
@@ -11,10 +11,6 @@
 
 defined('_JEXEC') or die;
 
-use Joomla\CMS\Language\Multilanguage;
-use Joomla\CMS\Uri\Uri;
-use Joomla\CMS\Factory;
```

Comment:
Please re integrate these 3 lines. They are the ones that break this patch for articles